packages/compass-indexes/src/components/index-header/index-header.jsx

```diff
@@ -1,9 +1,14 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import { css } from '@mongodb-js/compass-components';
 import IndexHeaderColumn from '../index-header-column';
 
-import classnames from 'classnames';
-import styles from './index-header.module.less';
+const containerStyles = css({
+  textAlign: 'left',
+  'th': {
+    padding: '12px 24px',
```

Comment:
Also maybe here we could replace these hardcoded paddings? Like use spacing or something.